### PR TITLE
Merge addmm

### PR DESCRIPTION
**Pull request by kzheng9**
Merging cherry-picked addmm. See https://github.com/cornell-brg/hb-pytorch/pull/29

### CODE REVIEW

File: hammerblade/torch/kernel/kernel_addmm.cpp [hunk 1]
```diff
@@ -1,29 +1,157 @@
...
+        auto mat1 = BSGTensor<float>(_mat1);
+        auto mat2 = BSGTensor<float>(_mat2);
+        auto result = BSGTensor<float>(_result);
+        float beta = *_beta;
+        float alpha = *_alpha;
+
+        // Start profiling
+        bsg_cuda_print_stat_kernel_start();
+
+        // TODO: Implement addmm
```
Comment (yodada) [inline hammerblade/torch/kernel/kernel_addmm.cpp:33]: no longer a TODO I guess? ;)

**Review by yodada — APPROVED** [review]
I think it looks good! Make sure you try it on COSIM.

File: hammerblade/torch/kernel/kernel_addmm.cpp (hunk 1 above)
Comment (kzheng9) [reply hammerblade/torch/kernel/kernel_addmm.cpp:33]: you are right, haha!

**Comment by cbatten** [comment]
@kzheng9 great work! Your first approved pull request -- a great milestone!

**Pull request merged**